server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -921,4 +924,58 @@ public void onFailure(Exception e) {
         assertNotNull(response.getTask().getError());
         assertNull(response.getTask().getResponse());
     }
+
+    public void testStoreResultWithAllFields() throws Exception {
+        // given
+        CyclicBarrier b = new CyclicBarrier(2);
+        TaskResultsService resultsService = internalCluster().getInstance(TaskResultsService.class);
+
+        // when
+        resultsService.storeResult(
+            new TaskResult(
+                new TaskInfo(
+                    new TaskId("fake", 1),
+                    "test",
+                    "test",
```

Comment:
Since you're specifically checking a "test" string later, it's generally a good idea to have different strings for the arguments here.